pkg/network.go

```diff
@@ -257,6 +260,32 @@ type PublicConfig struct {
 	Domain string `json:"domain"`
 }
 
+type NodeFeatures struct {
```

Comment:
NodeFeatures should have been a slice of a Feature []Feature and that feature is a typed string that can be any of `zdb, zmachine, .. etc)